charts/backbeat/values.yaml

```diff
@@ -0,0 +1,35 @@
+# Default values for backbeat.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image:
+  repository: zenko/backbeat
+  tag: 0.1.4
```

Comment:
In case it's decided to put backbeat-api as a part of this unified chart, then well have to use the image with ```tag: 0.1.5```.

Reply:
Of course, if this lands, the `api` part would be part of this chart, as another `Deployment` (I guess). That may indeed imply the version of the image needs to be bumped, so be it :)